ruby-gem/bin/configure_apply

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+lib_path = File.expand_path('../lib', __dir__)
+$:.unshift(lib_path) if !$:.include?(lib_path)
+require 'configure'
+
+Configure.apply(true, File.expand_path(".configure", Dir.pwd))
```

Comment:
Imho, it would feel nicer and more idiomatic to have a single binary, taking a subcommand to decide which of the 3 commands to run:

```suggestion
case ARGV.first
when 'init'
  Configure.init()
when 'apply'
  Configure.apply(true, File.expand_path('.configure', Dir.pwd)
when 'update'
  Configure.update(true, File.expand_path('.configure', Dir.pwd)
else
  puts "Unknown subcommand #{ARGV.first}."
  exit 1
end
```
(+ rename this file `ruby-gem/bin/configure`)

Reply:
This is a nice idea – added in a1f7fcfe8e41f1137a05edc6767424f473dffc86 (along with a bit of inline help)